oximeter/README.md

```diff
@@ -0,0 +1,184 @@
+# Oximeter
+
+The Oxide Rack metric collection system
+
+## Overview
+
+Oximeter is the system use to describe, generate, and store metrics and
+telemetry data in the Oxide Rack. The overall functionality is provided by a set
+of Rust crates:
+
+- [`oximeter`](1): The core crate, for describing and generating metric data
+  samples
+- [`collector`](2): The `oximeter` binary program run by the control plane,
+  which pulls metrics from other sources.
+- [`producer`](3): A library that allows programs to produce its metric data for
```

Comment:
Should this be:
A library that allows programs to produce their metric data for .. 
or
A library that allows a program to produce its metric data for ..

Reply:
The latter is probably easier to read, will update. Thanks!